package.json

```diff
@@ -38,6 +38,8 @@
     "ignore:rm": "lerna run ignore:rm",
     "lerna:ls:pl": "lerna ls -pl",
     "lint": "eslint '*/**/*.{js,ts,tsx}'",
+    "lint:scss": "stylelint 'packages/themes/*/src/*/*.scss' 'packages/components/**/*.scss'",
```

Comment:
prefer `packages/themes/*/src/**/*.scss` as the inside project structure may be more than one level